### ISSUE TITLE
Merge a pair of Agents by declaring one the source as the other the destination

### ISSUE THREAD

**Issue by dshorthouse** [AAFC-BICoE/dina-planning#221]
GIVEN I have accessed DINA as a collection manager

WHEN I come across a pair of Agents (= People)

THEN I should have the ability to declare one the source and the other the destination

AND then merge the two together

**********************************************

Like #220, this too may be a ropy feature but will likewise be necessary because Agents (= People) are meant to be unique objects, shared across collections. However, the source of these data will be insufficiently disambiguated. Unlike merging Collecting Events, merging Agent entries will be different because the names present in the source may be quite different than those in the intended destination. So, the source "names" should be added as aliases to the destination should these not already be present.

**Comment by cgendreau** [comment]
Note from discussion : support the concept of `same as` so the deleted agent could point to the new agent when requested. Deleted objects (soft-delete) will return an HTTP 410 Gone so we can include the link to the "real" agent.

**Comment by dshorthouse** [comment]
Am not convinced that the source Agent would/should return a http 410 if the intent is to also represent `sameAs`. Would not a http 301 be more appropriate?

**Comment by cgendreau** [comment]
The entity representing the source (uuid) is gone and the uuid of the preferred one would be returned to help the caller. They are 2 different things in the digital world. I see that differently as a 301 since the resource didn't move, it got replaced by something else.

**Comment by dshorthouse** [comment]
Fair enough, but the expectation of `sameAs` is broken if a GET on the destroyed source returns nothing but a `HTTP 410 Gone`. Any client hitting a destroyed resource would have no mechanism to make use of that `sameAs` to land on a destination.
